code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -11,15 +11,19 @@
 	origin_tech = "combat=2"
 	attack_verb = list("beaten")
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 50, BIO = 0, RAD = 0, FIRE = 80, ACID = 80)
-	/// How many life ticks does the stun last for
-	var/stunforce = 14 SECONDS
+	/// How many seconds does the knockdown last for?
+	var/knockdown_duration = 10 SECONDS
+	/// how much stamina damage does this baton do?
+	var/stam_damage = 60 // 2 hits or 1 hit + 2 disabler shots
 	/// Is the baton currently turned on
 	var/turned_on = FALSE
 	/// How much power does it cost to stun someone
 	var/hitcost = 1000
-	/// Chance for the baton to stun when thrown at someone
-	var/throw_hit_chance = 35
 	var/obj/item/stock_parts/cell/high/cell = null
+	/// the amount of time between swings
+	var/cooldown = 3.5 SECONDS
+	/// the time it takes before the target falls over
+	var/time_to_knockdown = 2.5 SECONDS
```

Comment:
ultra-nitpick, but wdyt renaming this var to `knockdown_delay` ?

Reply:
for you I will. (its a better var name)